ammo-be/src/webSocket/index.ts

```diff
@@ -3,7 +3,15 @@ import http from 'http';
 
 let ioServer: io.Server;
 
-let isRecording = false;
+// Bullets sub statuts
+let isSubbedToBullets = false;
+const getIsSubbedToBullets = (): boolean => isSubbedToBullets;
+
+// Disconnect callbacks. Can be useful for testing
```

Comment:
Good point. So first, it's technically not restricted to testing, it could be used for any purpose (it's generic enough), and second, I couldn't find any other way. If you have one we can get rid of this